### DISCUSSION TITLE
@formily/antd index 入口文件为什么要自动导入所有 style 文件呢？

### DISCUSSION THREAD

**Discussion by liuweiGL** [alibaba/formily#1482] [Q&A 2.x] [answered]
## 问题

antd 包的 index 自动导入了 style 文件，导致样式重复加载（比如使用 antd 的其他主题包）。现在为了避免这个问题，只能 `import xxx from @formily/antd/es/xxx` 放弃从入口文件导入。

## 建议

这个 style 可以打成一个 formily-antd.css/less 文件让用户选择是否要引入所有的样式，也可以让用户自己配置 babel 按需加载。

**Comment by janryWang** [comment, marked as answer]
为啥会重复打包，less会做去重的，如果你希望按需引入样式，可以配置ts-import-plugin

**Reply by liuweiGL** [reply]
嗯，那我再看看是不是编译配置问题
